modules/programs/antidote.nix

```
@@ -1,5 +1,6 @@
-{ config, lib, pkgs, ... }:
+{ config, lib, pkgs ? import <nixpkgs> { }, ... }:
```

Absolutely no `import <nixpkgs> {}` allowed here, there lie dragons 🙂

```suggestion
{ config, lib, pkgs, ... }:
```